docs/model_endpoint_management.ipynb

```diff
@@ -0,0 +1,621 @@
+    "* model_provider (str) : The provider of the model endpoint. Set to google for Vertex AI model endpoints and custom for custom-hosted model endpoints.\n",
+    "* model_type (str) : The model type. You can set this value to text_embedding for text embedding model endpoints or generic for all other model endpoints.\n",
+    "* model_qualified_name (str) : The fully qualified name in case the model endpoint has multiple versions or if the model endpoint defines it.\n",
+    "* model_auth_type (Optional[str]) : The authentication type used by the model endpoint. You can set it to either alloydb_service_agent_iam for Vertex AI models or secret_manager for other providers.\n",
+    "* model_auth_id (Optional[str]) : The secret ID that you set and is subsequently used when registering a model endpoint.\n",
+    "* input_transform_fn (Optional[str]) : The SQL function name to transform input of the corresponding prediction function to the model-specific input.\n",
+    "* output_transform_fn (Optional[str]) : The SQL function name to transform model specific output to the prediction function output.\n",
+    "\n",
+    "See below for an example of AlloyDBModel instance on using `aget_model(model_id=\"textembedding-gecko@001\")`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "AlloyDBModel(\n",
+    "    model_id=\"textembedding-gecko@001\",\n",
+    "    model_request_url=\"publishers/google/models/textembedding-gecko@001\",\n",
+    "    model_provider=\"google\",\n",
+    "    model_type=\"text_embedding\",\n",
+    "    model_qualified_name=\"textembedding-gecko@001\",\n",
+    "    model_auth_type=\"alloydb_service_agent_iam\",\n",
+    "    model_auth_id=None,\n",
+    "    input_transform_fn=\"google_ml.vertexai_text_embedding_input_transform\",\n",
+    "    output_transform_fn=\"google_ml.vertexai_text_embedding_output_transform\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Generate Vector Embeddings with `AlloyDBEmbeddings`\n",
+    "The `AlloyDBEmbeddings` class allows users to utilize the AlloyDB Embeddings available via Model Endpoint Management.\n",
+    "\n",
+    "In the below example, we are using the `textembedding-gecko@003` model that we have created using the Model Manager."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Note**: This tutorial demonstrates the async interface. All async methods have corresponding sync methods."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_google_alloydb_pg import AlloyDBEmbeddings\n",
+    "\n",
+    "model_id = \"textembedding-gecko@003\"\n",
```

Comment:
Either add disclaimer to recreate model since it would be dropped, or use `textembedding-gecko@001`